src/molecule/test/conftest.py

```diff
@@ -144,3 +145,13 @@ def reset_pytest_vars(monkeypatch):
     for var_name in tuple(os.environ):
         if var_name.startswith("PYTEST_"):
             monkeypatch.delenv(var_name, raising=False)
+
+
+@pytest.fixture(autouse=True)
+def block_on_serial_mark(request):
+    # https://github.com/pytest-dev/pytest-xdist/issues/385
+    if request.node.get_closest_marker("serial"):
+        with FileLock("semaphore.lock"):
```

Comment:
Instead of hardcoding this lock file in a project root, plz use a `tmp_path` fixture (maybe do this in a wrapper fixture with a session scope).

Reply:
We now use .tox folder and that is unique (opposed to tmp_path).